src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -25,35 +25,33 @@
               :else :same)
         tpos (+ pos (min nref nalt))
         d (Math/abs (- nref nalt))]
-    (when (and (not (= 1 nref nalt))
-               (not= 1 (count exon-ranges))
-               (some (fn [[s e]]
-                       (and (not= s e)
-                            (or (and (< pos s) (<= s (+ pos nref -1)))
-                                (and (<= pos e) (< e (+ pos nref -1))))))
-                     exon-ranges))
-      (throw
-       (ex-info
-        "Variants overlapping a boundary of exon/intron are unsupported"
-        {:exon-ranges exon-ranges, :pos pos, :ref ref, :alt alt})))
-    (->> exon-ranges
-         (keep (fn [[s e]]
-                 (case typ
-                   :ins (cond
-                          (< tpos s) [(+ s d) (+ e d)]
-                          (<= s tpos e) [s (+ e d)]
-                          :else [s e])
-                   :del (let [dels tpos
-                              dele (dec (+ tpos d))]
-                          (cond
-                            (< dele s) [(- s d) (- e d)]
-                            (<= dels s) (when (< dele e) [dels (- e d)])
-                            (<= dels e) (if (< dele e)
-                                          [s (- e d)]
-                                          [s (dec dels)])
-                            :else [s e]))
-                   :same [s e])))
-         vec)))
+    (if (and (not (= 1 nref nalt))
+             (not= 1 (count exon-ranges))
+             (some (fn [[s e]]
+                     (and (not= s e)
+                          (or (and (< pos s) (<= s (+ pos nref -1)))
+                              (and (<= pos e) (< e (+ pos nref -1))))))
+                   exon-ranges))
+      (do (log/warn "Variants overlapping a boundary of exon/intron are unsupported")
+          nil)
```

Comment:
I fix line 35-36 and change from `when` to `if` in line 28.
And I just fix indent of other lines.